pkg/download/downloader_test.go

```diff
@@ -444,3 +452,37 @@ func Test_extractArchive(t *testing.T) {
 		})
 	}
 }
+
+func TestTrueVerifier(t *testing.T) {
```

Comment:
This test was moved in order to co-locate it with the type definition.

Reply:
Tbh this test doesn’t make sense. 
Now that trueVerifier is a test-only type, we don’t need a test for this anymore.

Reply:
Agreed :)